livekit-plugins/livekit-plugins-silero/livekit/plugins/silero/onnx_model.py

```diff
@@ -0,0 +1,66 @@
+import atexit
+import importlib.resources
+from contextlib import ExitStack
+
+import numpy as np
+import onnxruntime  # type: ignore
+
+_resource_files = ExitStack()
+atexit.register(_resource_files.close)
+
+
+SUPPORTED_SAMPLE_RATES = [8000, 16000]
+
+
+def new_inference_session(force_cpu: bool) -> onnxruntime.InferenceSession:
```

Comment:
Should this be async? I think loading the weights for the first time takes a while

Reply:
I don't think we should, prewarming stuff will allow blocking ops